lib/stringify.js

```diff
@@ -24,10 +24,16 @@ var defaults = {
         return toISO.call(date);
     },
     skipNulls: false,
-    strictNullHandling: false
+    strictNullHandling: false,
+    formatters : {
+        RFC3986 : function (value) { return value; },
+        /** @return {string} */
+        RFC1738 : function (value) { return value.toString().replace(/%20/g, '+'); }
```

Comment:
let's use `String(value)` here - also, what happens if `value` is null or undefined?